docs/user-guide/ze-profiles.md

```diff
@@ -2,10 +2,119 @@
 
 After you install Zowe Explorer, you need to have a Zowe Explorer profile to use all functions of the extension. You can optionally activate the Secure Credential Store plug-in to securely store your credentials.
 
-**Note:** You can continue using Zowe V1 profiles with the Zowe Explorer V2. For more information, see ***insert link here***.
+**Note:** You can continue using Zowe V1 profiles with Zowe Explorer V2. 
+## Configuring team profiles
+
+Zowe CLI team profiles simplify profile management by letting you to edit, store, and share mainframe configuration details in one location. You can use a text editor or an IDE to populate configuration files with connection details for your mainframe services. By default, your team configuration file is located in the `.zowe home` folder, whereas the project-level configuration file is located in the main directory of your project.
+
+**Note**: A project context takes precedence over global configuration.
```

Comment:
Should we mention that it's possible to use the global configuration as long as the profile names are different ?